.goreleaser.yml

```diff
@@ -0,0 +1,55 @@
+# This is an goreleaser.yaml file which builds mongodb_exporter releases.
+# Make sure to check the documentation at http://goreleaser.com
+# You can run it locally or with your CI, but you should use `make community-release` or `make snapshot`
+# instead of using `goreleaser` directly as some init steps should be done before it runs.
+dist: bin
+builds:
+  - binary: mongodb_exporter
+    env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+      - darwin
+      - netbsd
+    goarch:
+      - 386
+      - amd64
+      - arm
+      - arm64
+    ignore:
+      - goos: darwin
+        goarch: arm
+      - goos: netbsd
+        goarch: arm
+      - goos: darwin
+        goarch: arm64
+      - goos: netbsd
+        goarch: arm64
+    flags:
+      - -v
+      - -a
+      - -tags=netgo
```

Comment:
We don't need "-a" and "-tags=netgo". Those a rudiments that a dragged for years from script to script, but the reality is Go don't need them to build everything properly for at least 3 years.